packages/portal-proto/src/features/proteinpaint/MatrixWrapper.tsx

```diff
@@ -218,7 +219,22 @@ export const MatrixWrapper: FC<PpProps> = (props: PpProps) => {
           // in case of race condition
           return prevData.current != data;
         },
-      });
+      })
+        .then?.((_app) => {
+          toolApp.current = _app;
+        })
+        .catch((e) => {
+          throw e;
```

Comment:
I am concerned about throwing an error here, as I am unsure how it will be handled.  We don't throw unhandled exceptions in the portal-prototype code.

Reply:
converted to `console.error()` instead of throw, see comments in my comment, this catch is not likely to be used except for bundling issues that are not detected during build